packages/samples/headless-react/src/components/smart-snippet/smart-snippet.fn.tsx

```diff
@@ -27,12 +28,68 @@ export const SmartSnippet: FunctionComponent<SmartSnippetProps> = (props) => {
     };
   };
 
-  const {answerFound, answer, question, liked, disliked, expanded} = state;
+  const {
+    answerFound,
+    answer,
+    question,
+    liked,
+    disliked,
+    expanded,
+    feedbackModalOpened,
```

Comment:
I think this should be `feedbackModalOpen` instead of opened... because opened is the verb.  what do you think? sorry I didn't see this before.